lrpar/src/lib/ctbuilder.rs

```diff
@@ -73,14 +74,58 @@ pub enum ActionKind {
     GenericParseTree
 }
 
+struct CTConflictsError<StorageT: Eq + Hash> {
+    pub grm: YaccGrammar<StorageT>,
+    pub sgraph: StateGraph<StorageT>,
+    pub stable: StateTable<StorageT>
+}
+
+impl<StorageT> fmt::Display for CTConflictsError<StorageT>
+where
+    StorageT: 'static + Debug + Hash + PrimInt + Serialize + TypeName + Unsigned,
+    usize: AsPrimitive<StorageT>,
+    u32: AsPrimitive<StorageT>
+{
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        let conflicts = self.stable.conflicts().unwrap();
```

Comment:
This `unwrap` seems a bit dodgy: if there are no conflicts it'll cause a panic I think?

Reply:
`CTConflictsError` only gets created if there are actual any errors, so `stable.conflicts` can not be `None` here.

Reply:
Aha!